tests/tck/features/go/GroupbyLimit.feature

```diff
@@ -323,6 +328,18 @@ Feature: Groupby & limit Sentence
       | "Dwyane Wade"     | 1.5 | 1     |
       | "Carmelo Anthony" | 1.5 | 1     |
 
+  @skip
```

Comment:
Why don't test group by

Reply:
It has bug and has been fixed in https://github.com/vesoft-inc/nebula-graph/pull/1229, so I delete it now